src/components/SingleMovie/SingleMovie.js

```diff
@@ -1,26 +1,38 @@
 import React from "react";
 import './SingleMovie.css'; 
 
-const SingleMovie = ({ image, title, releaseDate, overview, rating, genre, budget, revenue, runtime, tagline }) => {
+const SingleMovie = ({ singleMovie, handleBackButton }) => {
   
+	let total = singleMovie.runtime / 60;
+	let time = total.toString();
+	let minutes = time.substring(2, 4);
```

Comment:
Great work performing these actions before the return of the render - it keeps our render looking nice, clean, and easy to read!